code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -52,9 +67,10 @@
 	return ..()
 
 /obj/machinery/portable_atmospherics/process_atmos()
-	excited = (excited | air_contents.react(src))
-	if(!excited)
-		return PROCESS_KILL
+	if(!suppress_reactions)
+		excited = (excited | air_contents.react(src))
+		if(!excited)
```

Comment:
Not your fault since its also like that before, but ideally this should be outside of the supression if check. Telling the can to sleep should be independent of the crystal.